### PR TITLE
meta: move some background jobs to baseMeta

### PR DESCRIPTION
**Pull request by SandyXSD**

### CODE REVIEW

File: pkg/meta/tkv.go [hunk 1]
```diff
@@ -1717,38 +1684,21 @@ func (m *kvMeta) CopyFileRange(ctx Context, fin Ino, offIn uint64, fout Ino, off
...
-		last, err := m.get(m.counterKey("nextCleanupSlices"))
-		if err != nil || m.parseInt64(last)+3600 > now {
-			continue
-		}
-		_ = m.setValue(m.counterKey("nextCleanupSlices"), m.packInt64(now))
-		m.doCleanupSlices()
+	files := make(map[Ino]uint64, len(vals))
+	for k := range vals {
+		rb := utils.FromBuffer([]byte(k)[1:])
+		files[m.decodeInode(rb.Get(8))] = rb.Get64()
```
Comment (davies) [inline pkg/meta/tkv.go:1699]: the order between rb.Get(8) and rb.Get64() is not clear

File: pkg/meta/base.go [hunk 2]
```diff
@@ -127,6 +138,77 @@ func (r *baseMeta) newMsg(mid uint32, args ...interface{}) error {
...
+		m.Unlock()
+		if _, err := m.Load(); err != nil {
+			logger.Warnf("reload setting: %s", err)
+		}
+		go m.CleanStaleSessions()
+	}
+}
+
+func (m *baseMeta) CleanStaleSessions() {
+	sids, err := m.en.doFindStaleSessions(time.Now().Add(time.Minute * -5).Unix())
```
Comment (davies) [inline pkg/meta/base.go:202]: also this one (when there is lots of sustained file to clean up)

File: pkg/meta/base.go [hunk 3]
```diff
@@ -127,6 +138,77 @@ func (r *baseMeta) newMsg(mid uint32, args ...interface{}) error {
...
+		if m.umounting {
+			m.Unlock()
+			return
+		}
+		m.en.doRefreshSession()
+		m.Unlock()
+		if _, err := m.Load(); err != nil {
+			logger.Warnf("reload setting: %s", err)
+		}
+		go m.CleanStaleSessions()
```
Comment (davies) [inline pkg/meta/base.go:197]: if m.en.setIfSmall(...) {
  go m.CleanStaleSessions()
}

Comment (SandyXSD) [reply pkg/meta/base.go:197]: Will do in the next PR

**Pull request merged**